src/Scripts/Camera.gd

```diff
@@ -1,6 +1,9 @@
 extends Camera
 var zoom_factor = 1
-var zoom = 15.0
+
+export var min_zoom = 20.0
+export var max_zoom = 30.0
+onready var zoom = min_zoom
```

Comment:
zoom could be the middle between min_zoom and max_zoom: zoom = (min_zoom + max_zoom) / 2. 

This allows to the player have more flexibility with what he/she wants to see.